include/aspect/postprocess/velocity_residual_statistics.h

```diff
@@ -0,0 +1,137 @@
+/*
+  Copyright (C) 2014 - 2021 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+
+#ifndef _aspect_postprocess_velocity_residual_statistics_h
+#define _aspect_postprocess_velocity_residual_statistics_h
+
+#include <aspect/postprocess/interface.h>
+#include <aspect/simulator_access.h>
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+
+    /**
+     * A postprocessor that computes some statistics about the velocity residual in the model.
+     * The velocity residual is calculated as the difference between the modeled velocities and
+     * input ascii data velocities.
+     *
+     * @ingroup Postprocessing
+     */
+    template <int dim>
+    class VelocityResidualStatistics : public Interface<dim>, public ::aspect::SimulatorAccess<dim>
```

Comment:
Is this a good name, you think? It is a residual computed in a set of points, so maybe VelocityPointsResidual?

Reply:
I agree, it was a carry over from the previous version of this postprocessor. That makes sense, thank you!